auth/auth.go

```diff
@@ -442,14 +450,41 @@ func (auth *Authenticator) casUpdatePrincipal(p Principal, callback casUpdatePri
 }
 
 // Deletes a user/role.
-func (auth *Authenticator) Delete(p Principal) error {
-	if user, ok := p.(User); ok {
+func (auth *Authenticator) Delete(p Principal, purge bool, deleteSeq uint64) error {
+	user, ok := p.(User)
+	if ok {
```

Comment:
This doesn't look like it needed to be changed from the single line syntax it was using previously.